release/pkg/tasks/hashrelease.go

```diff
@@ -161,6 +162,11 @@ func HashreleaseValidate(cfg *config.Config, skipISS bool) {
 // CheckIfHashReleasePublished checks if the hashrelease has already been published.
 // If it has, the process is halted.
 func CheckIfHashReleasePublished(cfg *config.Config, hash string) {
+	if cfg.DocsHost == "" || cfg.DocsUser == "" || cfg.DocsKey == "" || cfg.DocsPort == "" {
+		logrus.Info("Missing hashrelease server configuration, skipping remote hashrelease check")
+		return
+	}
```

Comment:
i think this should error out unless it is a dev publish

Reply:
How would you suggest I determine this is a dev publish? I am using the presence of these fields to determine if we're running in the CI system or not.

Also, the existing logic doesn't error out if these fields are not present (it logs an error and continues).

Reply:
to check if we are running in a CI, you can check for the env variable `CI` so something like
```suggestion
	if cfg.DocsHost == "" || cfg.DocsUser == "" || cfg.DocsKey == "" || cfg.DocsPort == "" {
		log := logrus.Info
		if os.Getenv("CI") == "true" {
			log=logrus.fatal
		}
		log("Missing hashrelease server configuration, skipping remote hashrelease check")
		return
	}
```

Reply:
cool, thanks for the tip!